man/runc-exec.8.md

```diff
@@ -2,7 +2,7 @@
    runc exec - execute new process inside the container
 
 # SYNOPSIS
-   runc exec [command options] <container-id> <container command>
+   runc exec [command options] <container-id> <container command> [-- command options]
```

Comment:
I think this help string is just messed up, it should work the way you described.  @TristanCacqueray can you fix the usage strings?